docs/installation.rst

```diff
@@ -159,37 +159,29 @@ Django templates example:
 
 .. code-block:: html+django
 
-   <html>
-     <body>
-       {% if user.is_authenticated %}
-         <p>Current user: {{ user.email }}</p>
-         <form action="{% url 'oidc_logout' %}" method="post">
-           <input type="submit" value="logout">
-         </form>
-       {% else %}
-         <a href="{% url 'oidc_authentication_init' %}">Login</a>
-       {% endif %}
-     </body>
-   </html>
-
+   {% if user.is_authenticated %}
+     <p>Current user: {{ user.email }}</p>
+     <form action="{% url 'oidc_logout' %}" method="post">
+       {% csrf_token %}
+       <input type="submit" value="logout">
+     </form>
+   {% else %}
+     <a href="{% url 'oidc_authentication_init' %}">Login</a>
+   {% endif %}
 
 Jinja2 templates example:
 
 .. code-block:: html+jinja
 
-   <html>
-     <body>
-       {% if user.is_authenticated() %}
-         <p>Current user: {{ user.email }}</p>
-         <form action="{% url('oidc_logout') %}" method="post">
-           <input type="submit" value="logout">
-         </form>
-       {% else %}
-         <a href="{{ url('oidc_authentication_init') }}">Login</a>
-       {% endif %}
-     </body>
-   </html>
-
+   {% if request.user.is_authenticated() %}
```

Comment:
`is_authenticated` is a property in recent Django versions (since 1.10) and can not be called since the release of Django 2.0. Should I change this bit of the snippet as well?

Reply:
I would say yes. The snippet above has the property version so it will help us be consistent.